doc/recipe/preprocessor.rst

```diff
@@ -681,7 +680,7 @@ days in a year may vary between calendars, (sub-)daily data are not supported.
 .. code-block:: yaml
 
     preprocessors:
-      multimodel_preprocessor:
+      multi_model_preprocessor:
```

Comment:
why change the preprocessor name?

Reply:
For consistency. Some parts of the documentation used 'multimodel' and others 'multi-model'. So I just search and replaced all instances of multimodel. For the preprocessor name it doesn't really matter what it's called, but this now more consistent with the rest. Do you prefer multimodel without the hyphen?

Reply:
aha! good point! no, I don't give a hoots, but I was wondering why the change, all good by me then :+1: